.eslintrc.json

```
@@ -1,3 +1,47 @@
 {
-  "extends": ["next/core-web-vitals", "prettier"]
+  "extends": ["next/core-web-vitals", "prettier"],
+  "parserOptions": {
+    "sourceType": "module"
+  },
+  "env": { "es6": true },
+  "plugins": ["simple-import-sort"],
+  "rules": {
+    // Else Eslint will throw an error for `!!raw-loader!` imports
+    "import/no-webpack-loader-syntax": [0],
+    "simple-import-sort/imports": [
+      "error",
+      {
+        "groups": [
+          // Node.js builtins.
+          [
+            "^(assert|buffer|child_process|cluster|console|constants|crypto|dgram|dns|domain|events|fs|http|https|module|net|os|path|punycode|querystring|readline|repl|stream|string_decoder|sys|timers|tls|tty|url|util|vm|zlib|freelist|v8|process|async_hooks|http2|perf_hooks)(/.*|$)"
+          ],
+          // Packages. `react` related packages come first.
+          // Also, put `react-icons` in sorting order not with `react`
+          ["^react(?!-.)$", "^\\w", "^@\\w"],
+          // The Chakra theme directory if imported to story file or other places
+          ["^(\\.\\./)+@chakra-ui/gatsby-plugin/theme"],
```

This is different now but I'm wondering if it would be better to include the entire chakra folder just in case
```suggestion
          ["^(\\.\\./)+@chakra-ui"],
```